index.html

```diff
@@ -1751,10 +1597,12 @@ <h4>Additional information</h4>
 							<summary>Show / Hide additional information to understand this guideline and its success criteria.</summary>
 							<p class="subtitle"><strong>Benefits</strong></p>
 							<ul class="benefits">
+								<li><strong>Accessibility:</strong> Maintaining inclusivity over time through regular audits and testing reduces outages, improves access to information, and creates a better experience for all users, not just those with accessibility needs.</li>
```

Comment:
The accessibility benefit of collecting data about compliance is that it easier to track how accessibility developers over time. 

Saying inclusivity is confusing as it isn't the same.

I don't think tracking accessibility can lead to reduced outages. Accessible UIs are usually a better UI for all users but that's tangential to this I think? 

Suggestion to more clearly answer the question how does keeping compliance benefit accessibility:

```suggestion
								<li><strong>Accessibility:</strong> Tracking accessibility compliance information over time makes it easier for users to understand how acccessibility has been addressed in the past.</li>
```